javascript/workflow/src/resources/DataPackage/profiles/WorkflowProfile.ts

```diff
@@ -14,4 +14,18 @@ export class WorkflowProfile implements ProfileHandler<Workflow, WorkflowSchema>
 	public createInstance(schema: WorkflowSchema | undefined): Promise<Workflow> {
 		return Promise.resolve(new Workflow(schema))
 	}
+
+	public save(
+		data: Workflow,
+		dataPath: string,
+		files: Map<string, Blob>,
+	): Promise<string[]> {
+		const result: string[] = []
+		if (data.output?.table) {
+			const path = `${dataPath}output.csv`
```

Comment:
Maybe `${dataPath}${workflow.name ?? 'output'}.csv`?

Reply:
The downside here is that the workflow name is actually something like "workflow.json"

Reply:
Ok, latest commit fixes this by actually fixing a logic error I had previously made. The _workflow_ output is not what we want to save in the zip, it's the final _bundle_ output. So I moved the save function "up", and used the name, and the now matches the parent bundle name.